config/jobs/kubernetes-sigs/aws-iam-authenticator/aws-iam-authenticator-presubmits.yaml

```diff
@@ -37,3 +37,21 @@ presubmits:
       testgrid-tab-name: pull-integration
       description: AWS IAM Authenticator integration test on pull request
       testgrid-num-columns-recent: '30'
+  - name: pull-aws-iam-authenticator-e2e
+    always_run: true
+    decorate: true
+    labels:
+      preset-service-account: "true"
```

Comment:
You also need this one in order to get access to the aws testing account: `preset-aws-credential-aws-oss-testing: "true"`